lib/distro/distro.js

```diff
@@ -102,7 +103,9 @@ class Distro {
       // The lines including a library has the following format: ' => /path/to/library.so'
       if (line.match(' => /')) {
         const lib = line.match(/ => ([^\s]*)/)[1];
-        result.push(lib);
+        /* eslint-disable no-unused-expressions */
+        nfile.isLink(lib) ? result.push(fs.realpathSync(lib)) : result.push(lib);
```

Comment:
You wouldn't need the linter exception if you do this:
```suggestion
        lib = nfile.isLink(lib) ? fs.realpathSync(lib) : lib;
        result.push(lib);
```
note you need to change `const lib` to `let lib`

Reply:
thanks!